### ISSUE TITLE
Executed correctly, but no signatures generated

### ISSUE THREAD

**Issue by Apb58** [AlexandrovLab/SigProfilerExtractor#31]
Hello,

I am attempting to run sigProfilerExtractor on a set of input VCFs, and I believe everything is working as expected (no errors thrown that halt the process). However, upon inspection of the output directory, only the `SBS96` folder appears, and the contained `All_solutions_stat.csv` is empty. 

Here are my input commands and the returned output:
```
>>> from SigProfilerExtractor import sigpro as sig
>>> data = "/Users/Adrian/Desktop/TestSigProNewVer_vcfs"   # VCF files directory, 144 samples
>>> sig.sigProfilerExtractor("vcf", "TestSigProNewVer_out", data, minimum_signatures=1, maximum_signatures=10)

************** Reported Current Memory Use: 0.16 GB *****************

Starting matrix generation for SNVs and DINUCs...Completed! Elapsed time: 22.62 seconds.
Starting matrix generation for INDELs...Completed! Elapsed time: 6.39 seconds.
Matrices generated for 144 samples with 712919 errors. Total of 194855 SNVs, 14411 DINUCs, and 28350 INDELs were successfully analyzed.
Extracting signature 1 for mutation type 96
The matrix normalizig cutoff is 9600
```

Here is the `JOB_METADATA.txt` info:
```
THIS FILE CONTAINS THE METADATA ABOUT SYSTEM AND RUNTIME


-------System Info-------
Operating System Name: Darwin
Nodename: Adrians-MacBook-Pro-4.local
Release: 17.7.0
Version: Darwin Kernel Version 17.7.0: Sun Dec  1 19:19:56 PST 2019; root:xnu-4570.71.63~1/RELEASE_X86_64

-------Python and Package Versions-------
Python Version: 3.6.1
Sigproextractor Version: 1.0.12
SigprofilerPlotting Version: 1.1.6
SigprofilerMatrixGenerator Version: 1.1.16
Pandas version: 1.0.5
Numpy version: 1.19.0
Scipy version: 1.5.0
Scikit-learn version: 0.23.1

--------------EXECUTION PARAMETERS--------------
INPUT DATA
	input_type: vcf
	output: TestSigProNewVer_out
	input_data: /Users/Adrian/Desktop/TestSigProNewVer_vcfs
	reference_genome: GRCh37
	context_types: SBS96,DBS78,ID83
	exome: False
NMF REPLICATES
	minimum_signatures: 1
	maximum_signatures: 10
	NMF_replicates: 100
NMF ENGINE
	NMF_init: nndsvd_min
	precision: single
	matrix_normalization: gmm
	resample: True
	seeds: random
	min_NMF_iterations: 10,000
	max_NMF_iterations: 1,000,000
	NMF_test_conv: 10,000
	NMF_tolerance: 1e-15
CLUSTERING
	clustering_distance: cosine
EXECUTION
	cpu: 8; Maximum number of CPU is 8
	gpu: False
COSMIC MATCH
	opportunity_genome: GRCh37
	nnls_add_penalty: 0.05
	nnls_remove_penalty: 0.01
	initial_remove_penalty: 0.05
	refit_denovo_signatures: True

-------Analysis Progress-------
[2020-06-29 15:31:33] Analysis started:

##################################

[2020-06-29 15:32:11] Analysis started for SBS96. Matrix size [96 rows x 143 columns]

[2020-06-29 15:32:11] Normalization GMM with cutoff value set at 9600
```

Am I to interpret this as suggesting that no signatures were found?

Thanks,
Adrian

**Comment by mishugeb** [comment]
Hello,
Can you generate the matrix from your vcf files using SigProfilerMatrixGenerator (https://github.com/AlexandrovLab/SigProfilerMatrixGenerator)? 

Thanks

**Comment by lalexandrov1018** [comment]
Your matrices generated with many errors: "Matrices generated for 144 samples with 712919 errors." You are probably using the wrong reference genome. The parameters above specify "reference_genome: GRCh37". Is your data called against GRCh38?

**Comment by Apb58** [comment]
Yes, the data was generated with GRCh38 -- I did rerun again after updating the reference genome and the number of errors dropped to 3. However, I still was not getting an output folder with any results. I upgraded my python to 3.7.3 (see that I ran it initially with 3.6+) after reading through some of the other issues, and things seem to be working correctly, so it seemed the issue was linked to python. Just was strange for me to not encounter an error!
